docs/release/fundamentals/queries-and-expressions.ipynb

```diff
@@ -0,0 +1,2023 @@
+       "                <img src=\"data:image/webp;base64,UklGRsQAAABXRUJQVlA4ILgAAADwBACdASocABwAPmEojkWkIqEb/VQAQAYEtIACIJfpf53anfwh5bFxZyxmg4dIzIAA/v8aTg5Pf4AaLvDOb2+nBLe7h4WLP4eah5+VtiBuH5fnAK/9o/+/4nl+i/XuYMSDuGzWvPP7PeYAYqDelo/gGeaJWXI9qBrwbupRut1RmHAmjHeJdu952t5DneGb8d1NHD67m1X/4Fr/ktLUCjUDM3A9y/Tmv/u48GyVebp9QUFiS+qkAAAA\" width=\"28\" />\n",
+       "            </div></td>\n",
+       "      <td>{&quot;ground_truth&quot;: &quot;1&quot;, &quot;prediction&quot;: &quot;1&quot;, &quot;is_correct&quot;: true, &quot;engine&quot;: &quot;pixeltable&quot;}</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td><div class=\"pxt_image\" style=\"width:28px;\">\n",
+       "                <img src=\"data:image/webp;base64,UklGRhIBAABXRUJQVlA4IAYBAAAQBwCdASocABwAPmkqkEWkIqGYBgBABoS0gAiDrbQKlm9OvPM8p+ijvgH6ADs18j7y7UsovRG992j1lWJXI+NjsAD+/p9i31NdsRkVHtTBTbgEfcsiV5cS5IfF+yS7MDdvlaSHRre6nuYBWIGIMPNc7//PKE5FDkecaPH1C5KuSA/FmB0Mw15jQ4M0b5oct9Sv71vHA8P6IA5GrLvUA+xjJCnISNNaYSiHVONthArmojGAh1x2X9w6ER0Oe1Yqh//8NUVB//4iTpFaItcumEp4Ryr+jO63PtCFdy7qk9dlTU9Mno2GIXYCVLSAkv7l2L/rqd15c4R/MS1/O/u2yTOuN7uvmCQA\" width=\"28\" />\n",
+       "            </div></td>\n",
+       "      <td>{&quot;ground_truth&quot;: &quot;9&quot;, &quot;prediction&quot;: &quot;9&quot;, &quot;is_correct&quot;: true, &quot;engine&quot;: &quot;pixeltable&quot;}</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>"
+      ],
+      "text/plain": [
+       "                                               image  \\\n",
+       "0  <PIL.JpegImagePlugin.JpegImageFile image mode=...   \n",
+       "1  <PIL.JpegImagePlugin.JpegImageFile image mode=...   \n",
+       "2  <PIL.JpegImagePlugin.JpegImageFile image mode=...   \n",
+       "3  <PIL.JpegImagePlugin.JpegImageFile image mode=...   \n",
+       "4  <PIL.JpegImagePlugin.JpegImageFile image mode=...   \n",
+       "\n",
+       "                                               col_1  \n",
+       "0  {'ground_truth': '5', 'prediction': '5', 'is_c...  \n",
+       "1  {'ground_truth': '0', 'prediction': '0', 'is_c...  \n",
+       "2  {'ground_truth': '4', 'prediction': '4', 'is_c...  \n",
+       "3  {'ground_truth': '1', 'prediction': '1', 'is_c...  \n",
+       "4  {'ground_truth': '9', 'prediction': '9', 'is_c...  "
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "custom_dict = {\n",
+    "    # Keys must be strings; values can be any expressions\n",
+    "    'ground_truth': t.label,\n",
+    "    'prediction': t.pred_label,\n",
+    "    'is_correct': t.label == t.pred_label,\n",
+    "    # You can also use constants as values\n",
+    "    'engine': 'pixeltable',\n",
+    "}\n",
+    "\n",
+    "t.select(t.image, custom_dict).head(5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8377bb7f-f665-4ff9-a694-58c93ae28583",
+   "metadata": {},
+   "source": [
+    "### Function Calls\n",
```

Comment:
I'm wondering if 'Function Calls' is confusing and if UDF Calls would be better.